doc/conf.py

```
@@ -340,3 +401,30 @@
 intersphinx_mapping = {
     "python": ("https://docs.python.org/3", None),
 }
+
+
+# -- Suppress warnings for missing Doxygen XML ----------------------------
+# This allows building docs even when Doxygen hasn't been run
+
+def setup(app):
```

Ruff (0.14.10)

Unused function argument: `app`

(ARG001)